docs/package.nix

```
@@ -0,0 +1,138 @@
+{
+  ndg,
+  pkgs,
+  lib,
+}: let
+  inherit (builtins) isAttrs toString;
+  inherit (lib.attrsets) isDerivation mapAttrs optionalAttrs;
+  inherit (lib.modules) mkForce evalModules;
+  inherit (lib.options) mkOption;
+  inherit (lib.strings) hasPrefix removePrefix;
+  inherit (lib.trivial) pipe;
+
+  configJSON =
+    (pkgs.nixosOptionsDoc {
+      variablelistId = "hjem-options";
+      warningsAreErrors = true;
+
+      inherit
+        (
+          (evalModules {
+            modules =
+              [../modules/nixos/default.nix]
```

Pass in `self` into this file so we can write

```suggestion
              [self.nixosModules.hjem]
```